src/print/print.ts

```diff
@@ -21,5 +22,8 @@ export default async function print(
     options.forEach((arg) => args.push(arg));
   }
 
-  return execAsync(`lp ${args.join(" ")}`);
+  const {stdout: printResponse} = await execAsync(`lp ${args.join(" ")}`);
+  const jobId = getJobId(printResponse);	
+
+  return jobId;
```

Comment:
Hi @ppivanov, thank you very much for your contribution. I would like to see your work in master but I have a few remarks to consider.

Currently, the print function returns useful information in `{stdout, stderr}`. Would it be better to leave it as is and simply provide a new isPrintComplete function where the user can pass a print request id? For now, I think we can leave it to the user to parse the `'request ID is myDummyPrinter-4 (1 file(s))'` message. Additionally, will your `getJobId` function work if the printer name is `my-dummy-printer-4`?

I also think that the `isPrintComplete` function should accept the request id instead of the job id. This is because there might be many printers in a logistics department, for example, and it's possible that they might have the same job id at some point.

Reply:
> will your getJobId function work if the printer name is my-dummy-printer-4

I thought I took such cases into account, but it seems I only it was only a thought. It most definitely will not work as is!

We can most certainly change `getJobId` to return the request id instead.